lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -8,7 +8,11 @@
 		<div v-if="labelText || labelDataExists" class="progressBar__label">
 			<div
 				class="progressBar__labelText"
-				:class="{ '-medium': labelTextSize === PROGRESS_BAR_LABEL_TEXT_SIZES.MEDIUM }"
+				:class="{
+					'-medium': labelTextSize === PROGRESS_BAR_LABEL_TEXT_SIZES.MEDIUM,
+					'-ellipsis': labelTextEllipsis,
+				}"
+				:title="labelTextEllipsis ? labelText : ''"
```

Comment:
if you pass `null` or `undefined` (don't remember which) then Vue won't apply this attribute to HTML

Reply:
i checked and it can be null or undefined